### ISSUE TITLE
aria-hidden-focus rule is broken by TabsterDummyInput element

### ISSUE THREAD

**Issue by sarahhouse** [microsoft/tabster#288]
While testing our product that uses tabster, the Accessibility Insights fast pass failed because the aria-hidden-focus rule is being broken by one of tabster's components. Our component is a Toolbar that uses tabster to enable circular arrow navigation in the toolbar, and the element breaking this rule is a <i> that gets added above and below the children of the toolbar. 

According to the rule, elements that have "aria-hidden" = true should not be focusable. The <i> element has aria-hidden=true and tabindex=0, which are contradictory. Looks like this is being set here: https://github.com/microsoft/tabster/blob/3596de9c7029f9cfb3653c7a80956d37a22f007c/src/Utils.ts#L707. 

Link to the broken rule: https://accessibilityinsights.io/info-examples/web/aria-hidden-focus/

**Comment by mshoho** [comment]
@sarahhouse this particular issue should be added to the list of exceptions for Accessibility Insights — elements with `data-tabster-dummy` attribute are intentionally focusable with aria-hidden. Their only purpose is to redirect the focus to the proper element immediately after they receive focus. If they don't have aria-hidden, the screen readers will choke starting to announce them.

**Comment by MeghaY** [comment]
@mshoho Facing the same issue in the ReferenceListV2 component of Fluent 9. Is this added to the Accessibility Insights exception list. Any other resolution for this?
